lib/feature/pokemon_details_page.dart/pokemon_details_page.dart

```diff
@@ -78,13 +82,13 @@ class PokemonDetails extends StatelessWidget {
                                         : true,
                                     child: Container(
```

Comment:
Make a separate widget for this, so you can just reuse the widget wherever.

Reply:
line 80: 'pokemon.types?.last.type?.name == pokemon.types?.first.type?.name' already returns a boolean, so why make is a ternary?